nest/src/token/pUSD.sol

```diff
@@ -6,137 +6,493 @@ import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/I
 import { UUPSUpgradeable } from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 
 import { ERC20Upgradeable } from "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
-import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { ERC4626Upgradeable } from "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC4626Upgradeable.sol";
+import { ERC20 } from "@solmate/tokens/ERC20.sol";
 
-import { SafeTransferLib } from "@solmate/utils/SafeTransferLib.sol";
+import { IComponentToken } from "../interfaces/IComponentToken.sol";
 
-interface IVault {
+import { IAccountantWithRateProviders } from "../interfaces/IAccountantWithRateProviders.sol";
+import { IAtomicQueue } from "../interfaces/IAtomicQueue.sol";
 
-    function enter(address from, address asset, uint256 assetAmount, address to, uint256 shareAmount) external;
-    function exit(address to, address asset, uint256 assetAmount, address from, uint256 shareAmount) external;
-    function transferFrom(address from, address to, uint256 amount) external returns (bool);
-    function approve(address spender, uint256 amount) external returns (bool);
-    function balanceOf(
-        address account
-    ) external view returns (uint256);
+import { ILens } from "../interfaces/ILens.sol";
+import { ITeller } from "../interfaces/ITeller.sol";
+import { IVault } from "../interfaces/IVault.sol";
+import { IAccessControl } from "@openzeppelin/contracts/access/IAccessControl.sol";
+import { IERC20 } from "@openzeppelin/contracts/interfaces/IERC20.sol";
+import { IERC20Metadata } from "@openzeppelin/contracts/interfaces/IERC20Metadata.sol";
 
-}
+import { ReentrancyGuardUpgradeable } from "@openzeppelin/contracts-upgradeable/utils/ReentrancyGuardUpgradeable.sol";
+
+import { ComponentToken } from "../ComponentToken.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { Math } from "@openzeppelin/contracts/utils/math/Math.sol";
+import { FixedPointMathLib } from "@solmate/utils/FixedPointMathLib.sol";
```

Comment:
Could you format the imports so all the external imports are at the top and all the relative imports are at the bottom?